tests/test_models.py

```diff
@@ -30,7 +30,7 @@ def test_it_creates_a_job(webapp):
     context = Context(environment=test_env)
     context.set_config(url="http://localhost:8000", loop=30, retry=True)
 
-    schedule = Schedule(name="every two minutes", hour="*", minute="/2")
+    schedule = Schedule(name="every two minutes", hour="*", minute="*/2")
```

Comment:
can you also fix the demo script please?

Reply:
Resolved